### PR TITLE
Feat/update deps

### PR DESCRIPTION
**Pull request by realdreamer**
https://app.asana.com/0/1203822161704749/1205878510147258
https://app.asana.com/0/1203822161704749/1205878510147261
https://app.asana.com/0/1203822161704749/1205878510147256
https://app.asana.com/0/1203822161704749/1205878510147264
https://app.asana.com/0/1203822161704749/1205878510147266
https://app.asana.com/0/1203822161704749/1205886449630504
https://app.asana.com/0/1203822161704749/1205886449630505

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,13 +9,21 @@
...
   "bin": {
-    "create-kadena-app": "lib/index.js"
+    "create-kadena-app": "./bin/index.js"
   },
   "files": [
     "dist",
     "lib"
   ],
+  "contributors": [{
+    "name": "Jermaine de Jong",
```
Comment (jermaine150) [inline package.json]:
```suggestion
    "name": "Jermaine Jong",
```
Thanks for adding, small typo fix :-)

File: .prettierrc [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {
-  "printWidth": 80,
```
Comment (jessevanmuijden) [inline .prettierrc]: I thought Albert had strong reasons to keep it at 80.

File: templates/angular/.prettierrc.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc",
+  "printWidth": 120,
```
Comment (jessevanmuijden) [inline templates/angular/.prettierrc.json]: maybe 80

File: templates/vuejs/.prettierrc.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc",
+  "printWidth": 120,
```
Comment (jessevanmuijden) [inline templates/vuejs/.prettierrc.json]: maybe 80

File: templates/nextjs/src/pages/index.tsx [hunk 5]
```diff
@@ -1,91 +1,46 @@
...
-      const pollResult = await transactionBuilder.pollUntil(API_HOST, {
-        onPoll: async (transaction, pollRequest): Promise<void> => {
-          console.log(`Polling ${requestKey}.\nStatus: ${transaction.status}`);
-          console.log(await pollRequest);
-        },
+  function handleWriteMessageClick() {
+    setWriteInProgress(true);
+    writeMessage({ account, messageToWrite })
+      .then(() => {
+        setWriteInProgress(false);
```
Comment (jessevanmuijden) [inline templates/nextjs/src/pages/index.tsx]: could be moved to finally unless it absolutely needs to be called before `setMessageToWrite`. btw, why `then().catch()` when we can `async/await`?

**Review by jessevanmuijden — APPROVED** [review]

File: templates/nextjs/src/pages/index.tsx (hunk 5 above)
Comment (realdreamer) [reply templates/nextjs/src/pages/index.tsx]: not have any specific reason for promise catch. I think when i was following migration of kadena client examples, it's using catch so just followed along. thanks for bringing it up refactored to async/await

File: .prettierrc (hunk 2 above)
Comment (realdreamer) [reply .prettierrc]: Oh okay! I didn't knew this. I also checked other mono-repo it seems 80. so reverted back this change

**Review by jermaine150 — APPROVED** [review]

**Pull request merged**